src/util/TKUtil.h

```diff
@@ -129,4 +130,9 @@
  */
 + (NSString *)camelCaseToSnakeCase:(NSString *)string;
 
+/**
+ * Generates a random alias; useful for creating test members.
+ */
++ (Alias *)randomAlias;
+
```

Comment:
are you planning to expose this method to developers? If you are looking for a random Alias for the sample, you can use generateEmailAlias in TKTestBase

Reply:
Yes, planning to expose this method to developers.